stacker_blueprints/sns.py

```diff
@@ -53,17 +84,62 @@ def create_template(self):
         for topic_name, topic_config in variables["Topics"].iteritems():
             self.create_topic(topic_name, topic_config)
 
+    def create_sqs_policy(self, topic_name, topic_arn, topic_subs):
+        """
+        This method creates the SQS policy needed for an SNS subscription. It
+        also takes the ARN of the SQS queue and converts it to the URL needed
+        for the subscription, as that takes a URL rather than the ARN.
+        """
+        t = self.template
+
+        arn_endpoints = []
+        url_endpoints = []
+        for sub in topic_subs:
+            arn_endpoints.append(sub["Endpoint"])
+            split_endpoint = sub["Endpoint"].split(":")
+            queue_url = "https://%s.%s.amazonaws.com/%s/%s" % (
+                split_endpoint[2],  # literally "sqs"
+                split_endpoint[3],  # AWS region
+                split_endpoint[4],  # AWS ID
+                split_endpoint[5],  # Queue name
+            )
+            url_endpoints.append(queue_url)
+
+        policy_doc = queue_policy(topic_arn, arn_endpoints)
+
+        t.add_resource(
+            sqs.QueuePolicy(
+                topic_name + "SubPolicy",
+                PolicyDocument=policy_doc,
+                Queues=url_endpoints,
+            )
+        )
+
     def create_topic(self, topic_name, topic_config):
+        """
+        Creates the SNS topic, along with any subscriptions requested.
+        """
+        subscriptions = False
         t = self.template
 
+        if "Subscription" in topic_config:
+            subscriptions = True
+            topic_subs = topic_config["Subscription"]
+            topic_config["Subscription"] = process_subs(topic_subs)
+
         t.add_resource(
```

Comment:
I would try and replace this with:

```
        t.add_resource(
            sns.Topic.from_topic(
                topic_name,
                topic_config
            )
        )
```

This is new in the latest troposphere, and it should automatically do what you do in `process_subs` (ie: you can get rid of the if block above)